packages/web_ui/src/components/HostViewPage.tsx

```diff
@@ -34,9 +40,31 @@ export default function HostViewPage() {
 		</PageLayout>;
 	}
 
-	return <PageLayout nav={nav}>
-		<PageHeader title={host.name || String(hostId)} />
+	let hostButtons = <Space> {
+			account.hasAnyPermission("core.host.revoke_access", "core.admin") &&
```

Comment:
You do not need to check for "core.admin" when checking for another permission, it is special cased and having it makes all permission checks succeed.